controller/workitem_blackbox_test.go

```diff
@@ -776,6 +776,7 @@ func (s *WorkItem2Suite) TestWI2SuccessCreateWorkItem() {
 	assert.Equal(s.T(), "Title", wi.Data.Attributes[workitem.SystemTitle])
 	assert.NotNil(s.T(), wi.Data.Relationships.BaseType.Data.ID)
 	assert.NotNil(s.T(), wi.Data.Relationships.Comments.Links.Self)
+	//assert.NotNil(s.T(), wi.Data.Relationships.Area.Data.ID)
```

Comment:
Added it back since we needed it - had removed it temporarily to see the behaviour.
c211652ed546163b52ee89db97ae7ebf146e0681